libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/util/HttpUrlEx.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.navigation.route.internal.util
+
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+
+internal const val ACCESS_TOKEN_QUERY_PARAM = "access_token"
```

Comment:
add pls `todo move to mapbox-java`

Reply:
I'll follow up separately to not block the PR.